homeassistant/components/soundtouch/media_player.py

```diff
@@ -350,6 +352,21 @@ def play_media(self, media_type, media_id, **kwargs):
             else:
                 _LOGGER.warning("Unable to find preset with id %s", media_id)
 
+    def select_source(self, source):
+        """
+        Select the source to play audio from.
+
+        :param source:source to select
```

Comment:
Remove these lines because they are not necessary.

Reply:
Makes sense, since it's not a Soundtouch specific function